FileReader.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+)
+
+// check how stackoverflow works again
+// https://gosamples.dev/get-hostname-domain/
+func isValidURL(url_str string) bool {
```

Comment:
Everything looks good to me. Would this be a substitution for fileio/reader.go file you had previously? I would prefer having it the way you do now for functions regarding this area not being under CLI. You should go ahead and call your readFiles function from the cli/team23.go file.